modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,236 @@
+= APIkit for AsyncAPI Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit-asyncapi"]
+
+Learn the fundamentals of dependency, namespace, and schema information in APIkit for AsyncAPI to collaborate, build applications, and maintain codebases.
+
+== Dependency Information
+
+----
+<dependency>
+  <groupId>org.mule.modules</groupId>
+  <artifactId>mule-asynckit-module</artifactId>
+  <version>LATEST_VERSION</version> //<!--1-->
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+[calloutlist]
+.. Check https://docs.mulesoft.com/release-notes/apikit/apikit-release-notes[APIkit release notes] to get the *LATEST_VERSION*.
```

Comment:
Hard-coded http link? Should this be an xref?

Reply:
Yes! It should! Great catch! Thanks!